src/Catty/DataModel/Project/Project.m

```diff
@@ -299,6 +299,20 @@ + (nullable instancetype)projectWithLoadingInfo:(ProjectLoadingInfo*)loadingInfo
     NSDebug(@"%@", [project description]);
     NSDebug(@"ProjectResolution: width/height:  %f / %f", project.header.screenWidth.floatValue, project.header.screenHeight.floatValue);
     [self updateLastModificationTimeForProjectWithName:loadingInfo.visibleName projectID:loadingInfo.projectID];
+    
+    CBFileManager *fileManager = [[CBFileManager alloc] init];
+    NSString *defaultSceneDirectoryPath = [NSString stringWithFormat:@"%@%@", [project projectPath], [Util defaultSceneNameForSceneNumber:1]];
+    if (![fileManager directoryExists:defaultSceneDirectoryPath]) {
+        project.header.catrobatLanguageVersion = Util.catrobatLanguageVersion;
+        ProjectMigrator *migrator = [[ProjectMigrator alloc] initWithFileManager:fileManager];
+        NSError *error;
+        [migrator migrateToSceneWithProject:project error:&error];
+        
+        [migrator copyToRootFor:kScreenshotAutoFilename project:project error:&error];
```

Comment:
Please do that inside the migrator (the migrator might be reused in different places as well and needs to do everything in one place). Please also add a test verifying the copying of the screenshots.